### ISSUE TITLE
License/permission

### ISSUE THREAD

**Issue by parkus** [bendichter/brokenaxes#2]
I noticed there isn't one. I'd like to use this code for some plots I'll (hopefully) publish. Mind if I do? I'll add an acknowledgment to the paper. Saving me a a few hours of coding at least!

**Comment by bendichter** [comment]
Of course you can! Let me know if you need any help using it, and I'd love to see a draft when you are ready to share it.

**Comment by parkus** [comment]
Thanks!! Seems straightforward and cleanly coded. Happy to share the plot in question when it's done (hopefully in just an hour or two). The full paper, which addresses ultraviolet flares on low mass stars, won't be submitted for a few months at least.

**Comment by parkus** [comment]
Success! [fig_xoverlap.pdf](https://github.com/bendichter/brokenaxes/files/1114160/fig_xoverlap.pdf)

`brokenaxes` doesn't shine as much as it could in that figure since there is a lot of other stuff going on, but it solved my problem quite nicely. I imagine I will find many other uses for it. 

I had to make a few changes to `brokenaxes` to get what I wanted, but they were simple. I twiddled with `despine` because I wanted full axes boxes. Then I also needed to use `ax.errorbar` and not just `ax.plot` and since I might want to use many different plotting methods in the future I generalized the plotting behavior so that any call to an unrecognized attribute is turned into a call to all the subaxes. This might bite me later (perhaps it violates "explicit is better than implicit"), but I am pleased at the moment.

Thanks again!
